src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -11,7 +11,7 @@ export interface ResourceIF {
   addressLabel: string
   title?: string // BEN corrections only
   description?: string // BEN corrections only
-  filingData: FilingDataIF
+  filingData: FilingDataIF | Array<FilingDataIF>
```

Comment:
To support sub-types in the resource files (which are per filing type). This is the same thing I did in Create UI.